src/sdk/modules/validators/smartSessionValidator/tosmartSessionValidatorModule.ts

```diff
@@ -0,0 +1,118 @@
+import {
```

Comment:
Capital letter in the filename

Reply:
this is not easy repair now. either it fails on local or on github

Reply:
Just change it altogether to `toSmartSession` then? It's already in the `modules/validators` folder so it's overkill to name it this anyway.

Reply:
again, will have to change it for other modules in the same way. case (tosmartSession vs toSmartSession) in filename from previous commit history has been causing trouble